### PR TITLE
Update Dutch (NL_nl) translation

### PR DESCRIPTION
**Pull request by adrenth**
Update the Dutch translation of the current develop branch.

### CODE REVIEW

File: modules/backend/lang/nl/lang.php [hunk 1]
```diff
@@ -355,6 +361,8 @@
         'permissions' => 'De map :name of de submappen zijn niet schrijfbaar voor PHP. Zet de bijhorende rechten voor de webserver in deze map.',
         'extension' => 'De PHP extensie :name is niet geïnstalleerd. Installeer deze bibliotheek en activeer de extensie.',
         'plugin_missing' => 'De plugin :name is een afhankelijkheid maar is niet geïnstalleerd. Installeer deze plugin a.u.b.',
+        'debug' => 'Debug modus is ingeschakeld. Dit wordt niet aanbevolen op een productie-omgeving.',
+        'decompileBackendAssets' => 'Assets van de back-end worden momenteel  gedecompileerd. Dit wordt aanbevolen op een productie-omgeving.',
```
Comment (jacobdekeizer) [inline modules/backend/lang/nl/lang.php]: Remove double space

Comment (w20k) [reply modules/backend/lang/nl/lang.php]: That was fast 👍

Comment (jacobdekeizer) [inline modules/backend/lang/nl/lang.php]: Remove double space between "momenteel" and "gedecompileerd" and change the last sentence to "Dit wordt niet aanbevolen op een productie-omgeving."

Comment (w20k) [reply modules/backend/lang/nl/lang.php]: @jacobdekeizer please don't play `an admin`, even though the comment was not related to PR.
But I was about to add a comment for removing double `space`. Thank you!

**Comment by w20k** [comment]
@adrenth made need changes, I'll merge.

**Comment by adrenth** [comment]
Hi @w20k, I have removed the duplicate space.

**Comment by w20k** [comment]
Hi @adrenth, thanks. Done!

File: modules/backend/lang/nl/lang.php (hunk 1 above)
Comment (jacobdekeizer) [reply modules/backend/lang/nl/lang.php]: @w20k Sorry, I will write reviews differently in the future :)

File: modules/system/lang/nl/lang.php [hunk 2]
```diff
@@ -53,11 +53,6 @@
     'file' => [
         'create_fail' => 'Bestand aanmaken mislukt: :name',
     ],
-    'page' => [
-        'invalid_token' => [
-            'label' => 'Ongeldig token',
-        ],
-    ],
```
Comment (LukeTowers) [inline modules/system/lang/nl/lang.php]: Oy, I just did a lot of work to add that in, please undo this segment.

Comment (LukeTowers) [reply modules/system/lang/nl/lang.php]: nvm, I see @w20k already merged it. 👀

**Pull request merged**